crates/node-services/proto/schema.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2024 - Restate Software, Inc., Restate GmbH
+//
+// This file is part of the Restate service protocol, which is
+// released under the MIT license.
+//
+// You can find a copy of the license in file LICENSE in the root
+// directory of this repository or package, or at
+// https://github.com/restatedev/proto/blob/main/LICENSE
+
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package dev.restate.schema;
+
+service Schema {
+  // Fetch the current schema information
+  rpc FetchSchemas(FetchSchemasRequest) returns (FetchSchemasResponse);
+}
+
+message FetchSchemasRequest {
+  // provide schema version information
+}
+
+message FetchSchemasResponse {
+  // todo: Replace with proper protobuf
+  bytes schemas = 1;
```

Comment:
Can we pick a different name to make it easy to migrate in the future, maybe `schemas_bin`?

Reply:
Yes, good idea. I will change it.